src/V8JsiRuntime.cpp

```diff
@@ -452,7 +452,7 @@ std::once_flag g_tracingInitialized;
 
 void V8Runtime::initializeTracing() {
 #ifdef _WIN32
-  std::call_once(g_tracingInitialized, [](){ globalInitializeTracing(); });
+  std::call_once(g_tracingInitialized, []() { globalInitializeTracing(); });

```

Comment:
Looks like clang-format picked up a lot of fixes

Reply:
These are just a few lines. I do not think that it is an issue. Though, we usually prefer all such formatting changes going into a separate PR.

Reply:
okay good to know! I'll leave it then and make sure to note for next time :)